src/App.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'; 
+import './App.css';
+import chip from './images/IMG_4933.JPG';
+import rosie from './images/IMG_4932.JPG';
+import insta_logo from './images/insta_logo.png';
+import fb_logo from './images/fb_logo.png';
+import penguin from './images/penguin.png';
+import Navbar from './components/Navbar/Navbar';
+function App() {
+  const growPenguin = () => {
+    var penguin1 = document.getElementById('penguin1')
+    penguin1.style.transform = 'scale(1.1)'
+    penguin1.style.transition = "transform 0.25s ease"
+    document.getElementById("penguinButton").style.display = "none"
+  }
+  const onChangeFunction = (event) => {
+    var text = event.target.value
+    document.getElementById('penguinName').innerHTML = "This penguin's name is " + text
+  }
+  const foodList = ["bacon", "pizza", "cookie", "fries"]
+  const foodMap = foodList.map((food) => {
+    return(
+      <p>I like to eat <i className={'em em-'+ food} aria-role="presentation" aria-label={food}></i></p>
+    ) 
+  });
+  return (
+    <div className="background">
+        <Navbar />
+        <h1 className="synthwave">Hello! My name is Camden Hao and welcome to my website!</h1>
+        <link rel="stylesheet" href="style.css" />
+        <link href="https://emoji-css.afeld.me/emoji.css" rel="stylesheet" />
+        <p style={{fontSize: '20px'}} className="synthwave"><b>A little more about me: I'm a '23 from the DC/Northern Virginia area. 
+            I love playing chess, working out, and struggling to figure out which html tags do what. 
+            I've seen Friends and How I Met Your Mother 7 times each. My favorite book is <i>Kane and Abel
+            </i> by Jeffrey Archer. My favorite animal is the 
+            <i className="em em-penguin" aria-role="presentation" aria-label="BIRD" />
+            , and I have two dogs!</b></p>
+        <div className="flex-container">
+          <div>
+            <figure>
+              <img src={rosie} alt="This is Rosie!" className="dogImg"/>
```

Comment:
This image / fig caption duo can be a function! This will save the duplication in your code!